git/README.md

```diff
@@ -7,24 +7,53 @@ OCA maintainer quality tools.
 
 For using it, you can choose from two options:
 
-1. Copy files to local git repository into .git/hooks, and you will enable the
+1. Create symbolic link of files to local git repository into .git/hooks, and you will enable the
    checks for that git repository.
 
-2. Copy files to git template directory. In Ubuntu, it's located in:
+   ```bash
+   git clone git@github.com:OCA/maintainer-quality-tools.git ${HOME}/lint-hook
+   ln -sf ${HOME}/lint-hook/git/* {YOUR_PROJECT}/.git/hooks/.
+   ```
+
+2. Create symbolic link of files to git template directory.
+
+  In Ubuntu, it's located in:
+
+     /usr/share/git-core/templates/hooks/
+
+
+  In MAC, it's located in:
+
+    Installed from MAC-port
+
+        /opt/local/share/git-core/templates/hooks/
 
-   /usr/share/git-core/templates/hooks/
 
 Then, making `git init` inside a git repository, these files will be copied,
```

Comment:
I would recommend to make an bash-based alias to do that as the same command used in https://github.com/OCA/maintainer-quality-tools/pull/242/files#diff-eee548fe380dc59c94cc87563f0d454cR15 in the main directory as the repo is located, i.e.

```bash
alias install_git_hook='[[ -d .git ]] && ls -sf ${HOME}/lint-hook/git/pre-commit .git/hooks/pre-commit || echo "failed"'
```
IMHO this a user can be a handy alias to use this when is required in a existing repo